x/ibc/07-tendermint/types/store_test.go

```diff
@@ -0,0 +1,73 @@
+package types_test
+
+import (
+	"github.com/cosmos/cosmos-sdk/x/ibc/07-tendermint/types"
+	host "github.com/cosmos/cosmos-sdk/x/ibc/24-host"
+)
+
+func (suite *TendermintTestSuite) TestGetConsensusState() {
+	var (
+		height  uint64
+		clientA string
+	)
+
+	testCases := []struct {
+		name     string
+		malleate func()
+		expPass  bool
+	}{
+		{
+			"success", func() {}, true,
+		},
+		{
+			"consensus state not found", func() {
+				// use height with no consensus state set
+				height = height + 1
+			}, false,
+		},
+		{
+			"not a consensus state interface", func() {
+				// marshal an empty client state and set as consensus state
+				store := suite.chainA.App.IBCKeeper.ClientKeeper.ClientStore(suite.chainA.GetContext(), clientA)
+				clientStateBz := suite.chainA.App.IBCKeeper.ClientKeeper.MustMarshalClientState(&types.ClientState{})
+				store.Set(host.KeyConsensusState(height), clientStateBz)
+			}, false,
+		},
+		// TODO: uncomment upon merge of solomachine
```

Comment:
I wanted to add a test for the wrong consensus state concrete type being used but currently the tm consensus state is the only one so I will uncomment this when solomachine is merged